plugins/modules/aci_fabric_span_src_group_src_path.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Akini Ross <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_fabric_span_src_group_src_path
+short_description: Manage Fabric SPAN source paths (span:RsSrcToPathEp)
+description:
+- Manage Fabric SPAN source paths on Cisco ACI fabrics.
+options:
+  source_group:
+    description:
+    - The name of the Fabric SPAN source group.
+    type: str
+    aliases: [ name, src_group ]
+  source:
+    description:
+    - The name of the Fabric SPAN source.
+    type: str
+    aliases: [ name, src ]
+  pod:
+    description:
+    - The pod id of the source access path.
+    type: int
+    aliases: [ pod_id, pod_number ]
+  node:
+    description:
+    - The node id of the source access path.
+    type: int
+    aliases: [ node_id ]
+  path_ep:
+    description:
+    - The path of the source access path.
+    - An interface like C(eth1/7) must be provided.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+
+notes:
+- The C(source_group), and C(source) must exist before using this module in your playbook.
```

Comment:
```suggestion
- The I(source_group), and I(source) must exist before using this module in your playbook.
```

You mean the M(aci_fabric_span_src_group) and M(aci_fabric_span_src_group_src) objects should be present in the APIC?

Reply:
corrected